arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -56,7 +56,16 @@ An example of `chain.info-json` is available in the next section.
 --chain.name=<My Arbitrum L3 Chain>
 ```
 
-#### 3. AnyTrust chains
+
+#### 3. `execution.forwarding-target`
+
+`--execution.forwarding-target` you must set if you are running a normal fullnode (Not sequecner).
+
+```shell
+--execution.forwarding-targe=<Your Sequencer node endpoint url>
+```
+
```

Comment:
#### 3. `execution.forwarding-target`

you need to set `--execution.forwarding-target`  if you are running a normal full node (Not sequencer).
```shell
--execution.forwarding-targe=<Your Sequencer node endpoint URL>
```